superset/utils/core.py

```diff
@@ -1453,23 +1453,27 @@ def get_user_id() -> Optional[int]:
 
 
 @contextmanager
-def override_user(user: Optional[User]) -> Iterator[Any]:
+def override_user(user: Optional[User], force: bool = True) -> Iterator[Any]:
```

Comment:
The `force` flag was added for parity, however per my previous comment I'm not sure it's needed.